third_party/metrics_proto/chrome_user_metrics_extension.proto

```diff
@@ -66,22 +68,25 @@ message ChromeUserMetricsExtension {
 
   // This message will log one or more of the following event types:
   repeated UserActionEventProto user_action_event = 4;
-  repeated OmniboxEventProto omnibox_event = 5;
+  // Cobalt-only change to reduce compiled binary byte size. See b/290819695.
+  //repeated OmniboxEventProto omnibox_event = 5;
```

Comment:
This doesn't seem great to just comment it out but I can't think of a good alternative.

I don't think an extension (as suggested at https://yaqs.corp.google.com/eng/q/6275497799903608832#a1) would work for us.

Reply:
Ya, not really. Not without _significant_ changes throughout.

Reply:
I don't love these changes either, but i think literally commenting things out here is the best compromise probably.